app/src/common/constants.ts

```diff
@@ -51,8 +51,8 @@ export const TRADING_FEE_OPTIONS = [
   '5.00',
 ]
 
-export const GRAPH_MAINNET_HTTP = 'https://api.thegraph.com/subgraphs/name/protofire/omen'
-export const GRAPH_MAINNET_WS = 'wss://api.thegraph.com/subgraphs/name/protofire/omen'
+export const GRAPH_MAINNET_HTTP = 'https://api.thegraph.com/subgraphs/name/hilmarx/gelato-omen-mainnet'
+export const GRAPH_MAINNET_WS = 'wss://api.thegraph.com/subgraphs/name/hilmarx/gelato-omen-mainnet'
 export const GRAPH_RINKEBY_HTTP = 'https://api.thegraph.com/subgraphs/name/hilmarx/omen-gelato'
```

Comment:
Note: we changed the subgraph urls on mainnet and rinkeby to point to our deployed versions that include Gelato data, since our PR to omen-subgraph has not yet been merged. This is here so you can test everything on the UI properly now, once we merge the subgraph PR we'll change these urls back